hathor/transaction/resources/create_tx.py

```diff
@@ -0,0 +1,266 @@
+import base64
+from typing import Dict, List
+
+from twisted.web import resource
+
+from hathor.api_util import set_cors
+from hathor.cli.openapi_files.register import register_resource
+from hathor.crypto.util import decode_address
+from hathor.exception import InvalidNewTransaction
+from hathor.transaction import Transaction, TxInput, TxOutput
+from hathor.transaction.scripts import create_output_script
+from hathor.util import api_catch_exceptions, json_dumpb, json_loadb
+
+
+def from_raw_output(raw_output: Dict, tokens: List[bytes]) -> TxOutput:
+    value = raw_output['value']
+    token_uid = raw_output.get('token_uid')
+    if token_uid is not None:
+        if token_uid not in tokens:
+            tokens.append(token_uid)
+        token_data = tokens.index(token_uid) + 1
+    else:
+        token_data = 0
+    raw_script = raw_output.get('script')
+    if raw_script:
+        script = base64.b64decode(raw_script)
+    else:
+        address = decode_address(raw_output['address'])
+        script = create_output_script(address)
+    return TxOutput(value, script, token_data)
+
+
+@register_resource
+class CreateTxResource(resource.Resource):
+    """ Implements a web server API that receives hex form of a tx and returns the object
```

Comment:
It's the opposite, right? Receives an object and returns hexadecimal.

Reply:
Good catch. I copied DecodeTx, and that's its description. I'll fix it.

Reply:
Fixed.